src/Pages/HomePage/Homepage.jsx

```diff
@@ -18,25 +21,12 @@ const Homepage = () => {
 
   const addNoteHandler = async (event, note) => {
```

Comment:
You can create a separate file for all the API related functions & then import them here for Homepage.jsx to look cleaner